documentation/assemblies/assembly-external-logging.adoc

```diff
@@ -0,0 +1,36 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration.adoc
+
+[id='external-logging_{context}']
+= External logging
+
+When setting the logging levels for a resource, you can specify them _inline_ directly in the `spec` of the resource YAML:
+
+[source,shell,subs="+quotes,attributes"]
+----
+spec:
+  # ...
+  logging:
+    type: inline
+    loggers:
+      kafka.root.logger: "INFO"
+----
+
+Or you can specify _external_ logging:
+
+[source,shell,subs="+quotes,attributes"]
+----
+spec:
+  # ...
+  logging:
+    type: external
+    name: _customConfigMap_
+----
+
+With external logging, logging properties are defined in a ConfigMap.
+The name of the ConfigMap is referenced in the `spec`.
```

Comment:
do we want to be more precise saying `spec.logging.name` property/field?

Reply:
I've changed if you think it helps understanding